examples/basic/annotations/scale_bar_positioning.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+
+from bokeh.core.properties import field
+from bokeh.io import show
+from bokeh.layouts import column
+from bokeh.models import (ColumnDataSource, FactorRange, Metric,
+                          Range1d, ScaleBar, WheelZoomTool)
+from bokeh.palettes import Category10
+from bokeh.plotting import figure
+
+n_eeg_channels = 3
+n_pos_channels = 2
+n_channels = n_eeg_channels + n_pos_channels
+n_seconds = 15
+total_samples = 512*n_seconds
```

Comment:
```suggestion
total_samples = 512 * n_seconds
```

Reply:
It's a more common style, in my opinion, to have spaces around additive operators and no spaces around multiplicative.